index.js

```diff
@@ -4,15 +4,46 @@ console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
+header.style.color = "red"
 
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+const profile = document.querySelector("#profile")
+const name = profile.children[1];
```

Comment:
This works. You could also do 
travelerCard.querySelector('h2') 
this would make it more dynamic and allow you to change the order of the profile attributes without needing to change how you declare the variable